www/index.html

```diff
@@ -789,7 +789,14 @@ <h3 data-i18n="configure-expert-settings-title">Expert settings</h3>
                         <span id="alertMessage"></span>
                     </div>
                 </div>
-                <div id="navigationButtons" class="btn-group btn-block">
+                <div id="navigationButtons" class="btn-group btn-block" style="padding-left: 5%;">
+                    <div class="dropup" >
+                        <a href="#" class="btn btn-default dropdown-toggle col-xs-4" role="button" id="dropup" data-toggle="dropdown" aria-haspopup="true" aria-expanded="True" style="font-size: 14px; padding-top: 12px;">
```

Comment:
I think it's because you have copied code in the HTML from Kiwix PWA, whereas this Repo is using s different bootstrap. Look carefully at how the other buttons are wrapped, and copy it. For a start, the `<a ... >` container for the button here should have a class of `btn btn-lg`, rather than btn-default. Also, try to use the class-based shorthand for the button, don't copy and past the SVG code inline.